.envrc

```diff
@@ -6,3 +6,11 @@ export PUBLISHER_ID="0x7bd8a9757894069470bdee4d899b34b41ed82cbd77f68ba33d0fd46d0
 export USER1_PHRASE="gossip net bamboo assist danger armed soda plastic such father better grid"
 export USER2_PHRASE="can gaze pistol praise unable man afraid include route doll evoke ticket"
 export USER3_PHRASE="inherit chat inmate swamp transfer deposit shy heart frost chest cage bonus"
+export ADMIN_CAP_ID="0xa7785dd1cf60421de8e1a9dea7910e7e5615f25e26bebf7610c77aa9663a604f"
```

Comment:
@eivindingebrigtsen looks like the build just keeps appending to this file. Should we:

1. Add it to `.gitignore`
2. Make sure it doesn't append but overwrite